draft-ietf-quic-http.md

```diff
@@ -583,12 +673,34 @@ server to client, as in HTTP/2.  It defines no flags.
 
 The payload consists of:
 
-  Promised Stream ID:
-  : A 32-bit Stream ID indicating the QUIC stream on which the response will be
-    sent
+Push ID:
+: A 32-bit identifier for the server push request.  A push ID is used in push
+  stream header ({{server-push}}), CANCEL_PUSH frames ({{frame-cancel-push}}),
+  and PRIORITY frames ({{frame-priority}}).
+
+Header Block:
+: HPACK-compressed request headers for the promised response.
 
-  Payload:
-  : HPACK-compressed request headers for the promised response.
+A server MAY use the same Push ID in multiple PUSH_PROMISE frames.  This allows
+the server to use the same server push in response to multiple concurrent
+requests.  Referencing the same server push ensures that a PUSH_PROMISE can be
```

Comment:
Technically, there's nothing stopping you from having two different pushes for the same resource in flight at once.  I'm drawing a blank on why you would do that, but I'm not sure we want to foreclose it here.